dspace-jspui/src/main/java/org/dspace/app/webui/util/UIUtil.java

```diff
@@ -184,6 +195,78 @@ public static Context obtainContext(HttpServletRequest request)
 
         return c;
     }
+    
+     /**
+     * Returns a string array containing the URL to address this Item in DSpace,
+     * the official URL of its preferred identifier (example given
+     * http://dx.doi.org/10.5072/123) and the preferred identifier in canonical
+     * form (example given doi:10.5072/123). The configuration property 
+     * webui.preferred.identifier can be used to configure which identifier 
+     * should be preferred.
+     * If no identifier is found this method returns null. If no handle but a
+     * DOI is found the first value of the array is null.
+     * 
+     * @param ctx DSpace Context
+     * @param item the item
+     * @return 
+     * @throws SQLException
+     */
+    public static String[] getItemIdentifier(Context ctx, Item item)
+            throws SQLException
+    {
+        initialize();
+        // look up the the version handle
+        String versionHandle = item.getHandle();
+
+        // lookup the version doi
+        String versionDOI = null;
+        try {
+            versionDOI = identifierService.lookup(ctx, item, DOI.class);
+        }
+        catch (Exception ex)
+        {
+            log.info("Unable to lookup DOI for Item " + item.getID()
```

Comment:
I actually noted that we do neither need the try-catch-block nor the logging. iderntifierService.lookup(...) doesn't throw an exception (anymore?).